rotkehlchen/tests/integration/test_premium.py

```diff
@@ -25,28 +27,36 @@
 )
 from rotkehlchen.utils.misc import ts_now
 
+if TYPE_CHECKING:
+    from rotkehlchen.rotkehlchen import Rotkehlchen
+
 
 @pytest.fixture(name='premium_remote_data')
 def fixture_load_remote_premium_data() -> bytes:
     remote_db_path = Path(__file__).resolve().parent.parent / 'data' / 'remote_encrypted_db.txt'  # noqa: E501
     with open(remote_db_path, 'rb') as f:
-        return f.read()
+        return base64.b64decode(f.read())
```

Comment:
This is just for compatibility with the old way of doing things, right? Can you write this in the fiuxture? And also adjust the fixture to specify that this is the deprecated way?